index.js

```diff
@@ -111,10 +115,18 @@ class Installer {
       )
       child.pending = BB.resolve()
       .then(() => extract.child(child.name, child, childPath, this.config))
-      .then(() => readJson(childPath, 'package.json'))
+      .then(() => readPkgJson(path.join(childPath, 'package.json')))
       .then(pkg => {
         return this.runScript('preinstall', pkg, childPath)
         .then(() => this.extractTree(child))
+        .then(() => binLink(pkg, childPath, false, {
+          ignoreScripts: this.config.lifecycleOpts.ignoreScripts,
```

Comment:
There’s a few more options needed here that get passed from bin-links to gentleFS

Reply:
I have them in my local copy at work. Will send Monday, or you can figure it out :-). At least one is: `name: cipm`